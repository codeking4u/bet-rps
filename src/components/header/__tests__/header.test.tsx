import renderer from "react-test-renderer";

import Header from "../header.component";

test("matches snapshot", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
