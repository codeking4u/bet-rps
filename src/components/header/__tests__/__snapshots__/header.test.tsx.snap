// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="header"
>
  <div>
    Balance: 
    <span>
      5000
    </span>
  </div>
  <div>
    BET: 
    <span>
      0
    </span>
  </div>
  <div>
    WIN: 
    <span>
      0
    </span>
  </div>
</div>
`;
